lib/beaker/answers/version40.rb

```diff
@@ -0,0 +1,40 @@
+require 'beaker/answers/version34'
+
+module Beaker
+  # This class provides answer file information for PE version 4.0
+  #
+  # @api private
+  class Version40 < Version34
+    def generate_answers
+      masterless = @options[:masterless]
+      return super if masterless
+
+      master = only_host_with_role(@hosts, 'master')
+
+      the_answers = super
+
+      # remove some old answers
+      # - q_puppet_cloud_install
+      # - q_puppet_enterpriseconsole_database_name 
+      # - q_puppet_enterpriseconsole_database_password 
+      # - q_puppet_enterpriseconsole_database_user
+
+      the_answers.map do |vm, as|
+        as.delete_if do |key, value|
+          key =~ /q_puppet_cloud_install|q_puppet_enterpriseconsole_database_name|q_puppet_enterpriseconsole_database_password|q_puppet_enterpriseconsole_database_user/
```

Comment:
As noted in the ticket, I don't think the `q_puppet_enterpriseconsole_database_*` should be removed until we are ready in a couple weeks or a month. Otherwise this will break installs and we will have no way to fix it since it relies on external teams to deliver us new packages.